src/Microsoft.ML.FastTree/Representation/TreeRegressor.cs

```diff
@@ -0,0 +1,169 @@
+﻿using System;
+using System.Linq;
+using Microsoft.ML.Trainers.FastTree.Internal;
+
+namespace Microsoft.ML.FastTree.Representation
```

Comment:
>Microsoft.ML.FastTree.Representation [](start = 10, length = 36)

I do not like this namespace. Someone is just working with `FastTree`, why are they suddenly hit with this thing "representation?"

Reply:
Let me remove the `Representation`.

---
In reply to: [252498988](https://github.com/dotnet/machinelearning/pull/2243#discussion_r252498988) [](ancestors = 252498988)